README.developers.md

```diff
@@ -992,6 +992,56 @@ For example, the following will re-build only VPR using 8 parallel jobs with IPO
 make CMAKE_PARAMS="-DVTR_IPO_BUILD=off" -j8 vpr
 ```
 
+# Profiling VTR
+
+1. Install `gprof`, `gprof2dot`, and `xdot`. Specifically, the previous two packages require python3, and you should install the last one with `sudo apt install` for all the dependencies you will need for visualizing your profile results.
+    ```
+    pip3 install gprof
+    pip3 install gprof2dot
+    sudo apt install xdot
+    ```
+
+    Contact your administrator if you do not have the `sudo` rights.
+
+2. Enable profiling option when building vpr.
+    
+    You can enable profiling by editing the file `CMAKEList.txt` under `$VTR_ROOT`. Compile vpr after adding these two lines:
+    ```
+    set(PROFILING_FLAGS "-g -pg")
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pg")
+    ```
+    
```

Comment:
I'd suggest removing this, as the officially supported way is to set the CMake option as described below.